torch/csrc/jit/range_utils.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <algorithm>
+#include <cstddef>
+#include <cstdint>
+#include <utility>
+
+namespace torch {
+namespace jit {
+// Convert an python index (which may be negative) into an index usable for a
+// C++ container
+inline size_t normalizeIndex(int64_t idx, size_t list_size) {
```

Comment:
A `normalizeIndex` exists in `register_prim_ops.cpp` too, we should delete it and replace it with this